atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/TableMetadata.java

```diff
@@ -294,7 +295,7 @@ public int hashCode() {
         result = prime * result + (negativeLookups? 0 : 1);
         result = prime * result + (sweepStrategy.hashCode());
         result = prime * result + (appendHeavyAndReadLight? 0 : 1);
-        result = prime * result + (nameLoggable? 0 : 1);
+        result = prime * result + nameLogSafety.hashCode(); // Nonnull, because it has a default value
```

Comment:
Argh, all of the lines seem to have the excess brackets around them, probably because they've been auto-generated. For now, I'm going to standardise on having the excess brackets